src/components/CustomStatusBar/index.android.js

```diff
@@ -9,10 +8,11 @@ import themeColors from '../../styles/themes/default';
 export default class CustomStatusBar extends React.Component {
     componentDidMount() {
         StatusBar.setBarStyle('light-content');
-        StatusBar.setBackgroundColor(themeColors.appBG);
     }
 
     render() {
-        return <StatusBar />;
+        // Prefer to not render the StatusBar component in Android as it can cause
+        // issues with edge to edge display, which we setup in MainActivity.java.
+        return null;
```

Comment:
Why even import the library in that case? Lets use native android themes to do whatever `StatusBar.setBarStyle('light-content');` is doing